src/array/multipolygon/mutable.rs

```diff
@@ -64,6 +65,31 @@ impl<O: OffsetSizeTrait> MutableMultiPolygonArray<O> {
         }
     }
 
+    pub fn with_capacities_and_options(
```

Comment:
above, can you change `with_capacities` to delegate to `with_capacities_and_options` for reuse? And can you add a `new_with_options` here too? Thanks